lib/Wasmtime.hs

```diff
@@ -1029,8 +1043,11 @@ getMax mt = unsafePerformIO $
         else Just <$> peek max_ptr
 
 -- | Returns false if the memory is 32 bit and true if it is 64 bit.
-is64Memory :: MemoryType -> Bool
-is64Memory mt = unsafePerformIO $ withMemoryType mt c'wasmtime_memorytype_is64
+is64Memory' :: MemoryType -> Bool
+is64Memory' mt = unsafePerformIO $ withMemoryType mt c'wasmtime_memorytype_is64
+
+is64Memory :: MemoryType -> WordLength
+is64Memory mt = if is64Memory' mt then Bit64 else Bit32
```

Comment:
What about calling this one:
```suggestion
wordLength :: MemoryType -> WordLength
wordLength mt = if is64Memory' mt then Bit64 else Bit32
```
? And then removing the ' from `is64Memory'`.

Reply:
yes, but we are straying from the nomenclature of wasmtime..